States/InconsistentOpeningTimeEntryState.cs

```diff
@@ -4,12 +4,12 @@
 using FFXIVVenues.Veni.Context;
 using FFXIVVenues.Veni.States.Abstractions;
 using FFXIVVenues.Veni.Utils;
-using FFXIVVenues.VenueModels.V2022;
-using Venue = FFXIVVenues.Veni.Api.Models.Venue;
-
+using FFXIVVenues.VenueModels.V2022;
+using Venue = FFXIVVenues.Veni.Api.Models.Venue;
+
 namespace FFXIVVenues.Veni.States
 {
-    class InconsistentOpeningEntryState : IState
+    class InconsistentOpeningTimeEntryState : IState

```

Comment:
Renamed this class just so we can see all Time entry related states when we Ctrl + T for "TimeEntry" etc. since this state is working heavily with time.